.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,66 @@
+name: E2E tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  prettier:
+    name: Prettier
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          path: jest-light-runner
+
+      - name: Checkout "prettier"
+        uses: actions/checkout@v3
+        with:
+          repository: prettier/prettier
+          ref: next
+          path: prettier
+
+      - name: Run tests
+        run: |
+          cd prettier
+          yarn install --frozen-lockfile
+          yarn upgrade jest-light-runner@file:../jest-light-runner
+          yarn test:format
+
+  babel:
+    name: Babel
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          path: jest-light-runner
+
+      - name: Checkout "babel"
+        uses: actions/checkout@v3
+        with:
+          repository: babel/babel
+          path: babel
+
+      - name: Build
+        run: |
+          yarn install --frozen-lockfile
```

Comment:
`yarn install --immutable`?

Reply:
Uh yes, I wonder why Yarn didn't complain about the unknown option :thinking:

Reply:
On nevermind, it warns.

Reply:
65b7725bbfb262ec343000359ef87f345cfca5f9